Library/OcAppleKernelLib/CpuidPatches.c

```diff
@@ -981,6 +981,82 @@ mProvideCurrentCpuInfoZeroMsrThreadCoreCountPatch = {
   .Limit       = 0
 };
 
+// Offset of value in below patch.
```

Comment:
For proper Doxgen comment emission, please use
```
///
/// Offset of value in below patch.
///
```
Also please try to be clearer, this refers to the offset of the CoreCount argument within the instruction.